hadoop-project/pom.xml

```diff
@@ -184,9 +184,8 @@
     <surefire.fork.timeout>900</surefire.fork.timeout>
     <aws-java-sdk.version>1.12.367</aws-java-sdk.version>
     <hsqldb.version>2.7.1</hsqldb.version>
-    <aws-java-sdk-v2.version>2.19.12</aws-java-sdk-v2.version>
+    <aws-java-sdk-v2.version>2.20.128</aws-java-sdk-v2.version>
     <aws.evenstream.version>1.0.1</aws.evenstream.version>
```

Comment:
while you are here, how about renaming evenstream to eventstream; only just noticed this

Reply:
done